### PR TITLE
build for windows

### PR DESCRIPTION
**Pull request by Rechi**
I already build successfully and have tested in kodi.

Have only had an issue with an unresolved symbol at the last linking command. After I've replaced zlib with the precompiled version included in Kodi it worked.
Will investigate the problem further.

### CODE REVIEW

**Comment by notspiff** [comment]
looks all good, shall i merge now or do you want to fix the zlib issue first?

File: CMakeLists.txt [hunk 1]
```diff
@@ -21,7 +21,11 @@ include_directories(${INCLUDES}
                     ${KODI_INCLUDE_DIR})
                     
 list(APPEND DEPLIBS ${RTMP_LIBRARIES} ${OPENSSL_LIBRARIES} ${ZLIB_LIBRARIES})
-					
+
+if(CORE_SYSTEM_NAME MATCHES "windows")
+	list(APPEND DEPLIBS Winmm.lib WS2_32.lib)
```
Comment (notspiff) [inline CMakeLists.txt]: minor nit, indent seems to be a tab.

**Comment by notspiff** [comment]
maybe using the zlib definition from https://github.com/notspiff/audiodecoder.2sf/tree/master/depends/common/zlib takes care of it ?

File: CMakeLists.txt (hunk 1 above)
Comment (Rechi) [inline CMakeLists.txt]: yes is a tab, don't know how many spaces you expect
should I also change tab to spaces in `depends/windows/librtmp/CMakeLists.txt `?

Comment (notspiff) [inline CMakeLists.txt]: afaik the standard is 2 in kodi stuff.

Comment (notspiff) [inline CMakeLists.txt]: and yes, death to tabs, everywhere. you can of course format it nicely still but please use spaces.

**Comment by Rechi** [comment]
>looks all good, shall i merge now or do you want to fix the zlib issue first?

I don't know what was wrong yesterday, but I've retried today and it worked without any manual help.

**Comment by notspiff** [comment]
then in she goes.

**Comment by notspiff** [comment]
lol. i'm not thinking straight. the indent won't kill anyone.

**Comment by notspiff** [comment]
remember to active in repo-binary-addons (platforms.txt excludes windows)

**Comment by RoidosDiamond** [comment]
hey guys, so now the inputstream.rtmp works on windows?
if so, how can i install it? and keep it up to date?

**Comment by Rechi** [comment]
@RoidosDiamond it is included in the nightlies sine [today's nightly build](http://mirrors.kodi.tv/nightlies/win32/master/KodiSetup-20170217-128c5ba7ce-master.exe)
Hopefully it will be in tomorrows nightly build based on Krypton.
You can select in the installer which binary-addons you want to install. It also needs to be enabled in the addon manger in kodi.

**Comment by RoidosDiamond** [comment]
is there a way to add it to an existing installation of Krypton, like an addon?

**Comment by notspiff** [comment]
Please, do user support requests outside my mail box. There is ample info
on how to build in forums/wiki/git.

**Pull request merged**